src/friendly-iframe-embed.js

```diff
@@ -267,6 +268,12 @@ function mergeHtml(spec) {
     });
   }
 
+  // Load CSP
+  if (spec.cspEnabled) {
+    result.push('<meta http-equiv=Content-Security-Policy ' +
+      'content="script-src "none";object-src "none";child-src "none">');
```

Comment:
There are an odd number of double-quotes inside the angle brackets here, so something is not matched.
```
<meta http-equiv=Content-Security-Policy content="script-src "none";object-src "none";child-src "none">
```

Reply:
This is what caused your Travis failure, btw:
```
 ✗ AssertionError: expected '<html><head><base href="https://acme.org/embed1"><meta http-equiv=Content-Security-Policy content="script-src "none";object-src "none";child-src "none"></head><body></body></html>' to equal '<html><head><base href="https://acme.org/embed1"><meta http-equiv=Content-Security-Policy content="script-src \'none\';object-src \'none\';child-src \'none\'"></head><body></body></html>'
```

Reply:
Done

Reply:
Done